docker/docker-compose.yml

```diff
@@ -114,6 +114,10 @@ services:
       - "traefik.port=8080"
       - "traefik.protocol=https"
       - "traefik.docker.network=proxy"
+      # Redirect any requests to /uploads to the S3 container frontend.
+      - "traefik.uploads.frontend.redirect.regex=^https?://(?:.*?).altis.dev/uploads/(.*)"
```

Comment:
Just to check, this won't cause an infinite redirect right??

Reply:
I did test it 😛  - the traefik rules apply first before the request gets to the container so `s3-*.altis.dev` requests are never handled by nginx.

Reply:
WHen this says "redirect" does it mean HTTP redirect? If so, is this just used for legacy URLs? Just want to make sure we are not always issuing 301 redirects!